packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -105,6 +105,10 @@ export interface RemoteVideoStreamState {
    * Proxy of {@link @azure/communication-calling#RemoteVideoStream.isAvailable}.
    */
   isAvailable: boolean;
+  /**
+   * Proxy of {@link @azure/communication-calling#RemoteVideoStream.isReceiving}.
+   */
+  isReceiving: boolean;
```

Comment:
note: we will need to put this behind a beta conditional-compilation flag

Reply:
Ok ill add @beta. Do i just add the @beta tag here or everywhere?

Reply:
It's actually pretty complicated. We compile our codebase at all times for both stable and beta releases.
For stable we conditional-compile out all beta code.
We use compilation flags to achieve this, can read more here: https://github.com/Azure/communication-ui-library/blob/main/docs/references/beta-only-features.md
Have a stab if you like but otherwise I'm happy to add these flags once I get some time!

Reply:
I see, yep we also do this in web calling sdk.

Yes if you can pls add the beta flags, im not sure sure where to add and where not to add

Reply:
Actually looks like 1.5.4-stable is released! Will make more sense to simply update to that and not require the beta flags!
I'll throw up a PR to update our stable build to use the latest stable calling sdk package then circle back to this comment once that's completed

Reply:
Added beta flags 👍